src/services/formatting/formatting.ts

```diff
@@ -706,7 +706,6 @@ namespace ts.formatting {
                 if (isToken(child) && child.kind !== SyntaxKind.JsxText) {
                     // if child node is a token, it does not impact indentation, proceed it using parent indentation scope rules
                     const tokenInfo = formattingScanner.readTokenInfo(child);
-                    Debug.assert(tokenInfo.token.end === child.end, "Token end is child end");

```

Comment:
Was this assertion incorrect? This PR didn't change any parsing related things, AFAIK?